### PR TITLE
TWiR 347

### PR DESCRIPTION
**Pull request by cdmistman**

### CODE REVIEW

File: draft/2020-07-14-this-week-in-rust.md [hunk 1]
```diff
@@ -39,12 +40,27 @@ Check out [this week's *This Week in Rust Podcast*]()
...
+* [Subclassing GTK Widgets in Rust](https://www.figuiere.net/technotes/notes/tn002/)
 * [spanish] [Rust es orientado a objeto?](https://emanuelpeg.blogspot.com/2020/07/rust-es-orientado-objeto.html#.XwsegbMr_EQ.reddit)
 * [audio] [Fuzzing Rust with Shnatsel Podcast](https://medium.com/@social_62682/fuzzing-rust-with-shnatsel-podcast-e1fa0dbc28a)
 * [video] [Rust Loops Part 1: loop, break, continue](https://www.youtube.com/watch?v=zVX2qJFHDXA&feature=youtu.be)
 * [video] [Hitting A Bug in the Rust Compiler - While Writing A Boring Link Checker](https://www.youtube.com/watch?time_continue=4&v=DArJCR0HDL8&feature=emb_logo)
 * [video] [Two Sum Problem - Leet Code + Rust](https://www.youtube.com/watch?v=CMlHbAGkXjA&list=PLK_g1a_cAfaZuTXzDoQUAFEHCalKSCv9G&index=2)
 * [video] [Rust + WebAssembly - EdgeXR @ Netlight](https://www.youtube.com/watch?v=dmbqpg5BuBY)
+* [video] [Rust Meetup - Adding WASM Support to a Native Application (GameDev Edition)](https://www.youtube.com/watch?v=7YQGwb4_AvA)
+* [video] [Async Interview #8: Stjepan Glavina](http://smallcultfollowing.com/babysteps/blog/2020/07/09/async-interview-8-stjepan-glavina/)
```
Comment (nellshamrell) [inline draft/2020-07-14-this-week-in-rust.md:63]: I think we've covered this one up above with a link to the blog post. Would you want to link directly to the video, or should we figure that people can go to the blog post and find the video that way?

Comment (cdmistman) [reply draft/2020-07-14-this-week-in-rust.md:63]: I'll delete it, I added that link before updating my branch and missed the blog post

Comment (cdmistman) [reply draft/2020-07-14-this-week-in-rust.md]: fixed!

Comment (nellshamrell) [reply draft/2020-07-14-this-week-in-rust.md]: Yay!

**Comment by nellshamrell** [comment]
Thank you so much!

**Pull request merged**